docs/phase-8-foundation-cleanup.md

```
@@ -0,0 +1,1060 @@
+- [ ] `src/app/login/page.tsx` (theming)
+- [ ] `src/middleware.ts` (auth guards)
+- [ ] `.eslintrc.json` (new rules)
+- [ ] All Withings API routes (middleware)
+- [ ] `CLAUDE.md` (add patterns reference)
+- [ ] All files with `console.log`
+
+---
+
+## Change Log
+
+**Version 1.0 - January 2025**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)